docs/CHANGELOG.md

```diff
@@ -13,7 +13,12 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 
 * `amethyst_test` test framework for ergonomic testing of Amethyst applications ([#1000])
 
+### Changed
+
+* Replaced `GlobalTransform` with `Transform` in all examples. ([#1070])
```

Comment:
You probably don't need to mention that in the changelog because it's only a meta change.

Reply:
Got it. I wasn't exactly sure how big the changelog's scope was.

Reply:
Basically you need to add something to the changelog when you change a visible behavior (new features, bug fixes, API changes, etc...) in the engine itself. Here it's just the examples so it's fine.